set.mm

```diff
@@ -471236,6 +471248,84 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+       Arnoux, 15-Jan-2024.) $)
+    eqg0el $p |- ( ( G e. Grp /\ H e. ( SubGrp ` G ) ) -> ( [ X ] .~ = H <-> X
+       e. H ) ) $=
+      ( cgrp wcel csubg cfv wa cec wceq c0g wbr cbs wer eqid eqger adantl wb
+      grpidcl adantr erth eqgid eqeq1d eqcom a1i 3bitrrd wrel errel 3syl eleq2d
+      relelec 3bitr2d ) BFGZCBHIGZJZDAKZCLZBMIZDANZDUTAKZGZDCGUQVAVBURLCURLZUSU
+      QUTDABOIZUPVEAPZUOABVECVEQZERZSUOUTVEGUPVEBUTVGUTQZUAUBUCUQVBCURUPVBCLUOA
+      BVECUTVGEVIUDSZUEVDUSTUQCURUFUGUHUPVCVATZUOUPVFAUIVKVHVEAUJDUTAUMUKSUQVBC
+      DVJULUN $.
+  $}
+
+  ${
+    $d A x y $.
+    $( The quotient set of a cartesian product is trivial.  (Contributed by
+       Thierry Arnoux, 16-Jan-2024.) $)
+    qsxpid $p |- ( A =/= (/) -> ( A /. ( A X. A ) ) = { A } ) $=
+      ( vy vx c0 wne cxp cqs csn cv wceq wrex wcel wa simpr ecxpid adantr eqtrd
+      cec adantl wex rexlimiva n0 biimpi simpl eqtr4d ancld eximdv mpan9 df-rex
+      ex sylibr impbida vex elqs velsn 3bitr4g eqrdv ) ADEZBAAAFZGZAHZURBIZCIZU
+      SRZJZCAKZVBAJZVBUTLVBVALURVFVGVFVGURVEVGCAVCALZVEMZVBVDAVHVENVHVDAJZVEAVC
+      OZPQUASURVGMVICTZVFURVHCTZVGVLURVMCAUBUCVGVHVICVGVHVEVGVHVEVGVHMVBAVDVGVH
+      UDVHVJVGVKSUEUJUFUGUHVECAUIUKULCAVBUSBUMUNBAUOUPUQ $.
+  $}
+
+  ${
+    $d B x y $.  $d G x y $.  $d R x y $.
+    qustriv.1 $e |- B = ( Base ` G ) $.
+    $( The Group quotient equivalence relation for the whole group is the
+       cartesian product, i.e. all elements are in the same equivalence class.
+       (Contributed by Thierry Arnoux, 16-Jan-2024.) $)
+    qusxpid $p |- ( G e. Grp -> ( G ~QG B ) = ( B X. B ) ) $=
+      ( vx vy cgrp wcel cqg co cxp csubg cfv wer wrel eqid a1i cv wa wbr wb w3a
+      subgid eqger errel 3syl relxp cminusg cplusg simpl grpinvcl adantrr grpcl
+      simprr syl3anc pm4.71d df-3an syl6rbbr wss ssid eqgval mpan2 brxp 3bitr4d
+      ex eqbrrdv ) BFGZDEBAHIZAAJZVFABKLGAVGMVGNABCUBVGBAACVGOZUCAVGUDUEVHNVFAA
+      UFPVFDQZAGZEQZAGZVJBUGLZLZVLBUHLZIAGZUAZVKVMRZVJVLVGSZVJVLVHSZVFVSVSVQRVR
+      VFVSVQVFVSVQVFVSRVFVOAGZVMVQVFVSUIVFVKWBVMABVNVJCVNOZUJUKVFVKVMUMAVPBVOVL
+      CVPOZULUNVDUOVKVMVQUPUQVFAAURVTVRTAUSVJVLVPVGABVNFACWCWDVIUTVAWAVSTVFVJVL
+      AAVBPVCVE $.
+
+    $d B a x $.  $d G a x $.  $d Q a x $.
+    qustriv.2 $e |- Q = ( G /s ( G ~QG B ) ) $.
+    $( The quotient of a group ` G ` by itself is trivial.  (Contributed by
+       Thierry Arnoux, 15-Jan-2024.) $)
+    qustriv $p |- ( G e. Grp -> ( Base ` Q ) = { B } ) $=
+      ( cgrp wcel cqg co cqs cxp cbs cfv csn qusxpid qseq2d cvv cqus wceq a1i
+      ovexd id qusbas c0 wne grpbn0 qsxpid syl 3eqtr3d ) CFGZACAHIZJAAAKZJZBLMA
+      NZUJUKULAACDOPUJUKCBAQFBCUKRISUJETACLMSUJDTUJCAHUAUJUBUCUJAUDUEUMUNSACDUF
+      AUGUHUI $.
+  $}
+
+  ${
+    $d B x $.  $d G x $.  $d H x $.  $d Q x $.
+    qustrivr.1 $e |- B = ( Base ` G ) $.
+    qustrivr.2 $e |- Q = ( G /s ( G ~QG H ) ) $.
+    $( Reciproque of ~ qustriv .  (Contributed by Thierry Arnoux,
```

Comment:
Is reciproque an English word (it's definitively French). Would "reciprocal" be an adequate translation?

Reply:
I ended up using "converse", which I think is the correct term.